app/models/work.rb

```diff
@@ -2,7 +2,7 @@
 class Work < ApplicationRecord
   GENRES = %w(poetry prose drama fables article memoir letters reference lexicon).freeze
   
-  has_and_belongs_to_many :expressions
+  has_many :expressions, inverse_of: :work
```

Comment:
Don't we want dependent: :destroy on this relationship?  Deleting a work should mean deleting all its expressions, since we're not implementing compound expressions containing more than one work.

Reply:
Good catch, thanks. Fixed